Directory.Build.props

```diff
@@ -34,11 +35,12 @@
     <AndroidTargetFrameworkVersion>30.0</AndroidTargetFrameworkVersion>
     <WindowsTargetFrameworkVersion>10.0.19041</WindowsTargetFrameworkVersion>
     <Windows2TargetFrameworkVersion>10.0.20348</Windows2TargetFrameworkVersion>
+    <TizenTargetFrameworkVersion>6.5</TizenTargetFrameworkVersion>
   </PropertyGroup>
 
   <PropertyGroup>
     <!-- the real TFMs -->
-    <MauiPlatforms>net$(_MauiDotNetVersion)-ios;net$(_MauiDotNetVersion)-maccatalyst;net$(_MauiDotNetVersion)-android</MauiPlatforms>
+    <MauiPlatforms>net$(_MauiDotNetVersion)-ios;net$(_MauiDotNetVersion)-maccatalyst;net$(_MauiDotNetVersion)-android;net$(_MauiDotNetVersion)-tizen</MauiPlatforms>
```

Comment:
This looks like anyone building Maui would need the Tizen workload installed?

Should we make this optional? Because it would break a lot of contributors? Maybe it could just be something you uncomment to work on Tizen?

Reply:
Also, not sure how this plays with the fact that we don't have a way to install the tizen workload without using the script. So maybe we can have all the files in place, and maybe this can just be removed from the TFM list here?

Unless, there is a way to make .msi installers that can be pushed to nuget that just are a no op or something. That is probably for another issue with the dotnet sdk team, but I suppose the safest is to just remove the TFM from here for now until we have a non-script way to install workloads.

Also, this may actually require the insertion of the tizen workload into VS and I am not sure they will allow non-msi workloads. 

A bit annoying I am sure, but not sure there is another way at the moment.

Reply:
Ok, so maybe it need to be an MSBuild property that CI passes in?

And developers working on Tizen could uncomment something in `Directory.Build.props`?

Reply:
What do you think about adding this only in CI environment for release and build test?
In this way, I think contributors can develop as before without any break.

e.g.
```sh
<MauiPlatforms Condition="'$(CI)' == 'true'">$(MauiPlatforms);net$(_MauiDotNetVersion)-tizen</MauiPlatforms>
```

Reply:
I think it's better to add only when tizen workload is installed, but I can't come up with a good idea to check this through MSBuild. Any advices?

Reply:
One idea would be to put this in `Directory.Build.props`:
```xml
<Import Condition="Exists('Directory.Override.props')" Project="Directory.Override.props" />
```
Add this file to `.gitignore`, locally you could create this file, and set whatever properties you like.

So, for example, a property like `IncludeTizenTargetFrameworks=true` that is normally blank.